From 479eef923b756fdcfea395d63e2383f18479ada0 Mon Sep 17 00:00:00 2001
From: Michael Truog <mjtruog@protonmail.com>
Date: Sat, 18 May 2019 19:36:17 -0700
Subject: [PATCH] Add Erlang/OTP 23 compatibility.  Add OCaml 4.08.0
 compatibility.  Update the Erlang binary term format encoding implementations
 to support Erlang/OTP 23 with the programming languages Go, Haskell, Java,
 Javascript, OCaml, Perl, PHP, Python and Ruby.  Update uuid.

diff --git a/src/api/ocaml/cloudi.ml b/src/api/ocaml/cloudi.ml
index 044396dd..c661af37 100644
--- a/src/api/ocaml/cloudi.ml
+++ b/src/api/ocaml/cloudi.ml
@@ -162,7 +162,7 @@ module Instance = struct
   let callbacks_remove api pattern =
     let key = api.prefix ^ pattern in
     let value = Hashtbl.find api.callbacks key in
-    let _ = Queue.pop value in
+    let _f = Queue.pop value in
     if Queue.is_empty value then
       Hashtbl.remove api.callbacks key ;
     ()
